### PR TITLE
DPL-1104 - Adds NovaSeq 6000 PE sequencing request to the RVI Bait Capture Library Prep submission

### LINKED ISSUE #4023: DPL-1104 - Add the NovaSeq 6000 sequencing request to the RVI Bait Capture Library Prep submission
**User story**
As a user, Di R, I would like the RVI Bait Capture Library Prep submission to include a NovaSeq 6000 sequencing request. This will remove the need to 'request additional sequencing' in Sequencecape on the final pool tube of the RVI BCL pipeline.

**Who are the primary contacts for this story**
Ben T, Andrew S

**Who is the nominated tester for UAT**
Di R, Scott T

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] RVI BCL submission also creates the NovaSeq 6000 sequencing requests
- [ ] Within the RVI BCL pipeline 'Request additional sequencing' will no longer be required to set sequencing requests on RVI BCL final pool tubes.
- [ ] Update integration suite to add a check for sequencing requests presence at the end of the pipeline? See how other pipelines test sequencing requests e.g. 'Limber-Htp - ISC - NovaSeq 6000 Paired end sequencing'

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #4023 

#### Changes proposed in this pull request

- Adds NovaSeq 6000 PE sequencing request to the RVI Bait Capture Library Prep submission

#### Instructions for Reviewers

Requires manually updating
```
bundle exec rails c
rt = RequestType.find_by(key: 'illumina_htp_novaseq_6000_paired_end_sequencing')
st = SubmissionTemplate.find_by(name: 'Limber-Htp - RVI - Bait Capture Library')
st.name = 'Limber-Htp - RVI Bait Capture Library - NovaSeq 6000 Paired end sequencing'
st[:submission_parameters][:request_type_ids_list].push(rt.id)
st.save
```

### CODE REVIEW

**Review by andrewsparkes — APPROVED** [review]
Just check after deployment in the rails console or database that record loader has updated this.

File: config/default_records/submission_templates/010_rvi_bait_capture_library_submission_templates.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@
 Limber-Htp - RVI - Bait Capture Library:
```
Comment (KatyTaylor) [inline config/default_records/submission_templates/010_rvi_bait_capture_library_submission_templates.yml:4]: Just wondering whether the submission template name should be changed too. Normally, the submission template name includes the kind of sequencing.

Comment (BenTopping) [reply config/default_records/submission_templates/010_rvi_bait_capture_library_submission_templates.yml:4]: Good point. Ill contact the stakeholders and ask.

Comment (BenTopping) [reply config/default_records/submission_templates/010_rvi_bait_capture_library_submission_templates.yml]: Updated here and in the int suite.

**Review by KatyTaylor — APPROVED** [review]

**Pull request merged**